Dockerfile

```diff
@@ -0,0 +1,40 @@
+# debian/buster
+FROM php:7.3-cli
+
```

Comment:
Can we add some labels?
LABEL vendor="Phalcon"
LABEL maintainer="team@phalcon.io"